winui/Masked-TextBox/Getting-Started.md

```diff
@@ -7,4 +7,85 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Getting Started with WinUI SfMaskedTextBox
\ No newline at end of file
+# Getting Started with WinUI SfMaskedTextBox
+
+This section explains the steps required to add the WinUI MaskedTextBox control and covers the basic features needed to get started with Syncfusion `MaskedTextBox` control.
+
+## Creating an application with WinUI MaskedTextBox control
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/get-started-winui3-for-desktop).
+2. Add reference to [Syncfusion.Editors.WinUI](https://www.nuget.org/packages/Syncfusion.Editors.WinUI) NuGet. 
+3. Import the control namespace `Syncfusion.UI.Xaml.Editors` in XAML or C# code.
+4. Initialize the `MaskedTextBox` control.
+
+## Initialize MaskedTextBox control using Simple mask
+
```

Comment:
@Vignesh-SF3580 Initialize the MaskedTextBox control using simple mask

Reply:
Changed the sentence like above.